Jenkinsfile

```diff
@@ -40,30 +40,44 @@ pipeline {
             }
         }
 
-        stage('Static analysis') {
-            agent { label 'builder' }
-            steps {
-                deleteDir()
-                unstash "StaticAnalysis"
-                unstash "SourceCode"
-                unstash "CIScripts"
-                powershell script: "./StaticAnalysis/Invoke-StaticAnalysisTools.ps1 -RootDir . -Config ${pwd()}/StaticAnalysis"
-            }
-        }
+        stage('Static analysis ans tests') {
+            parallel {
+                stage('Static analysis on Windows') {
+                    agent { label 'builder' }
+                    steps {
+                        deleteDir()
+                        unstash "StaticAnalysis"
+                        unstash "SourceCode"
+                        unstash "CIScripts"
+                        powershell script: "./StaticAnalysis/Invoke-StaticAnalysisTools.ps1 -RootDir . -Config ${pwd()}/StaticAnalysis"
+                    }
+                }
 
-        stage('Linux-test') {
-            when { expression { env.ghprbPullId } }
-            agent { label 'linux' }
-            options {
-                timeout time: 5, unit: 'MINUTES'
-            }
-            steps {
-                deleteDir()
-                unstash "Monitoring"
-                dir("monitoring") {
-                    sh "python3 -m tests.monitoring_tests"
+                stage('Static analysis on Linux') {
+                    agent { label 'linux' }
+                    steps {
+                        deleteDir()
+                        unstash "StaticAnalysis"
+                        unstash "Ansible"
+                        sh "StaticAnalysis/ansible_linter.py"
+                    }
+                }
+
+                stage('Linux-test') {
```

Comment:
We think that it would be better to name that stage `CI test`. Right now it only runs the CI tests which require Linux. In the future we would similar thing of Windows CI tests.